src/main/java/games/strategy/engine/data/AllianceTracker.java

```diff
@@ -82,20 +82,20 @@ protected void addToAlliance(final PlayerID player, final String allianceName) {
    */
   public Set<PlayerID> getPlayersInAlliance(final String allianceName) {
     final Iterator<PlayerID> keys = alliances.keySet().iterator();
-    final HashSet<PlayerID> rVal = new HashSet<>();
+    final HashSet<PlayerID> playersInAlliance = new HashSet<>();
```

Comment:
Not sure if you want to adress this in this PR, but using the interface is sufficient in this case

Reply:
Done in e1ab290.